pages/api/augment-request.ts

```diff
@@ -69,7 +69,7 @@ async function getRequestTxFromL1RequestInformation(
   l1Requests.forEach((l1Request) => {
     let foundOoTx = { requestTransactionHash: "", chainId: 0, oracleType: "" };
     chainOoRequests.forEach((_, ooChainId) => {
-      chainOoRequests[ooChainId].forEach((ooRequest, index) => {
+      chainOoRequests[ooChainId].forEach((ooRequest) => {
```

Comment:
this is unrelated but i could not build, due to linting error. dont know how this passed ci in previous pr

Reply:
This now fixed 👍